pkg/apis/core/helper/helpers.go

```diff
@@ -186,7 +186,13 @@ func IsExtendedResourceName(name core.ResourceName) bool {
 // implicitly in the kubernetes.io/ namespace.
 func IsNativeResource(name core.ResourceName) bool {
 	return !strings.Contains(string(name), "/") ||
-		strings.Contains(string(name), core.ResourceDefaultNamespacePrefix)
+		IsPrefixedNativeResource(name)
+}
+
+// IsPrefixedNativeResource returns true if the resource name
+// has a kubernetes.io/ substring.
+func IsPrefixedNativeResource(name core.ResourceName) bool {
```

Comment:
"Native" doesn't add any meaning here, and actually makes it a bit harder to understand - does it just mean "cpu" and "memory"?  No, it's really `HasKubePrefix` or something?